### DISCUSSION TITLE
Disable multi line input

### DISCUSSION THREAD

**Discussion by obaid** [ueberdosis/tiptap#1691] [General Questions] [answered]
I am trying to find out how to disable multi line input? i.e. simply use it as a `input` field rather than `textarea`?

**Comment by jloganolson** [comment, marked as answer]
I think you can create a custom document extension that just supports a single block instead of multiple (so you can't do new lines):
```
import Document from "@tiptap/extension-document";

export default Document.extend({
  content: "block"
});
```
